synth.py

```diff
@@ -59,7 +59,7 @@
 # ----------------------------------------------------------------------------
 # Add templated files
 # ----------------------------------------------------------------------------
-templated_files = common.py_library(cov_level=100, samples=True)
+templated_files = common.py_library(cov_level=100, samples=True, split_system_tests=True)
 
 # BigQuery has a custom multiprocessing note
 s.move(templated_files, excludes=["noxfile.py", "docs/multiprocessing.rst"])
```

Comment:
I think we need to exclude `docs/conf.py` as well, to prevent future synth failures.

Reply:
I can do that, but I think we should rather fix this by modifying the template to include the excludes. WDYT?

Reply:
I updated the template in synthtool and re-generated the files. Now we don't manually modify the files.